src/wasm2js.h

```diff
@@ -424,13 +424,13 @@ Ref Wasm2JSBuilder::processWasm(Module* wasm, Name funcName) {
   });
   if (generateFetchHighBits) {
     Builder builder(allocator);
-    asmFunc[3]->push_back(processFunction(
-      wasm,
+    std::unique_ptr<Function> tempFunc(
```

Comment:
Since this `Function` only lives for a very short scope, I think we should be able to just stack allocate it, avoiding the `unique_ptr` (and even the `builder`) entirely.

Reply:
Opted for adding the function to the module now. `generateFetchHighBits` should already ensure that it isn't added twice iiuc. Also, there are other functions already being added if I'm not mistaken, like from i64 lowering, so this should be fine and one can inspect the module afterwards to see exactly what became converted if needed.